extensions/opentelemetry/opentelemetry-exporter-otlp/deployment/pom.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-opentelemetry-exporter-otlp-parent</artifactId>
+        <version>999-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>quarkus-opentelemetry-exporter-otlp-deployment</artifactId>
+    <name>Quarkus - OpenTelemetry Exporter - Otlp - Deployment</name>
```

Comment:
It's minor, but should "Otlp" be all capitalized? I usually see it in all capitals.